components/Card/Card.tsx

```diff
@@ -0,0 +1,47 @@
+import Link from "next/link";
+import getDateFormatted from "../../lib/date";
+import { IPost } from "../../lib/posts";
+import Avatar from "../Avatar/Avatar";
+import ImageWithPlaceholder from "../ImageWithPlaceholder/ImageWithPlaceholder";
+import styles from "./Card.module.scss";
+
+interface ICardProps {
+  post: IPost;
+}
+
+const Card = (props: ICardProps): JSX.Element => {
+  const { post } = props;
+  const { authorProfile, category, date, featuredImage, id, subtitle, title } = post;
+  return (
+    <Link href={`/post/${id}`}>
+      <a className={styles.link}>
+        <div className={styles.card}>
+          <div className={styles.header}>
+            <ImageWithPlaceholder
+              layout="fill"
+              objectFit="cover"
+              src={`/posts/featured/${featuredImage}`}
+            />
+          </div>
+          <div className={styles.details}>
+            <span className={styles.postDetails}>
+              <p className={styles.category}>{category}</p> {" - "}
```

Comment:
Hmm, guessing prettier didn't like this. Think this might come out as `[space][space][hyphen][space]`? 😢

Reply:
Looks like it just sacks off all the spaces anyway 🤷🏼‍♂️